src/components/icons.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+export const save = (
```

Comment:
Can you add a comment at the top with attribution of where you found those icons